### PR TITLE
Feature - #2130 - Auto-focuses on the Username field of Door43 and Guest Login

### PR DESCRIPTION
**Pull request by richmahn**
#### This pull request addresses:

Addresses the issue #2130 

#### How to test this pull request:

Logout (if you are already logged in) and then go to the login selection screen. Click on "Door43" and it should have the Username field selected. Go back and click to login as a guest. It should also have the Username field focused.

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3208)

### CODE REVIEW

**Comment by mannycolon** [comment]
@richmahn you need to merge with develop and the tools commits shouldn't be showing in your changes.

**Comment by mannycolon** [comment]
@richmahn tetss are failing and decreased

**Comment by mannycolon** [comment]
Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[tC_apps/translationNotes, line 1 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208#-Kz58OMSLA2cz8vTA0RI:-Kz58OMSLA2cz8vTA0RJ:bqly7lm) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/bbbac1c13c675500b9f99d308430f11048ad94b8/tC_apps/translationNotes#L1)):*
> ```
> Subproject commit ec7f6f734473679d8c6489a66c4c032c39ce7e21
> ```

remove submodules commits

---

*[tC_apps/translationWords, line 1 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208#-Kz58SpcWS7eHJgyKp66:-Kz58SpcWS7eHJgyKp67:bqly7lm) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/bbbac1c13c675500b9f99d308430f11048ad94b8/tC_apps/translationWords#L1)):*
> ```
> Subproject commit 5395a72c684cdd543a915aff9c26195399f3b2e6
> ```

remove submodules commits

---

*[tC_apps/VerseCheck, line 1 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208#-Kz58TMS8_8wmsFGtsS9:-Kz58TMS8_8wmsFGtsSA:bqly7lm) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/bbbac1c13c675500b9f99d308430f11048ad94b8/tC_apps/VerseCheck#L1)):*
> ```
> Subproject commit b44a509ec53a4674db771dc78c1efce68db753b7
> ```

remove submodules commits

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208)*

**Comment by mannycolon** [comment]
Reviewed 5 of 5 files at r2.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[__tests__/CreateLocalAccount.test.js, line 4 at r2](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208#-Kz5E_cFgPDAsq3wH3GK:-Kz5E_cFgPDAsq3wH3GL:bicfepp) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/30ec896314c55a0a16d7c8419a35df028fbe999f/__tests__/CreateLocalAccount.test.js#L4)):*
> ```JavaScript
> 
> import React from 'react';
> import CreateLocalAccount from '../src/js/components/home/usersManagement/LoginDoor43Account';
> ```

how about we rename this the same as the component name which in this case is LoginDoor43Account

---

*[__tests__/LoginDoor43Account.test.js, line 4 at r2](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208#-Kz5EtH8vFxSqt8ShpKf:-Kz5EtH8vFxSqt8ShpKg:b-b0ttwb) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/30ec896314c55a0a16d7c8419a35df028fbe999f/__tests__/LoginDoor43Account.test.js#L4)):*
> ```JavaScript
> 
> import React from 'react';
> import LoginDoor43Account from '../src/js/components/home/usersManagement/LoginDoor43Account';
> ```

In other words, this tow tests files should be together in the same file 

so CreateLocalAccount.tests.js and LoginDoor43Account.tests.js  should be together

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208)*

**Comment by mannycolon** [comment]
@richmahn tests are still decreasing codecov

**Comment by richmahn** [comment]
@mannycolon I was thinking unit test files should test different components. This changes two components, the LoginDoor43Account component and CreateLocalAccount component. Both of their code is independent.

**Comment by richmahn** [comment]
@mannycolon The CreateLocalAccount.test.js tests CreateLocalAccount. I just made the mistake of forgetting to rename the import file.

**Comment by mannycolon** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/3208)*
